pkg/cli/helpers.go

```diff
@@ -189,7 +189,7 @@ func watch(fn func(r sdk.Interface, c *stdcli.Context) error) func(sdk.Interface
 }
 
 func checkRackNameRegex(name string) error {
-	if !regexp.MustCompile(`^[a-z0-9-]+$`).MatchString(name) {
+	if !regexp.MustCompile(`^[a-z0-9/-]+$`).MatchString(name) {
```

Comment:
why are you allowing `/` in here?

Reply:
for installing console rack we use a combination of orgname/rackname.
As the '/' wasn't allowed so it would not install console rack from cli.

Reply:
this will allow rack name to have `/` which we don't allow, so remove this.
Also you shouldn't mark comment resolve, I will mark them resolved if they are resolved.

Reply:
I have updated the code with the changes, pls check.